app/store/chat.ts

```diff
@@ -92,6 +93,9 @@ function getSummarizeModel(currentModel: string) {
   if (currentModel.startsWith("gemini-pro")) {
     return GEMINI_SUMMARIZE_MODEL;
   }
+  if (currentModel.startsWith("claude")){
```

Comment:
currentlly,only internal Claude Models supported,so just identifying claude models by prefix 'claude' is not right

Reply:
Appreciation for the review! However, I was a little confused; Are 'internal Claude models' these?
```ts
const anthropicModels = [
  "claude-instant-1.2",
  "claude-2.0",
  "claude-2.1",
  "claude-3-sonnet-20240229",
  "claude-3-opus-20240229",
  "claude-3-haiku-20240307",
];
```
In that case, I didn't see identifying via prefix has any problem...  
If that means custom models (some users may be using reverse proxy for claude but with OpenAI API) should be supported as well, will using `includes` or regex be appropriate?

> `includes`

```ts
if (currentModel.toLowerCase().includes("claude") || currentModel.toLowerCase().includes("anthropic")){
    return ANTHROPIC_SUMMARIZE_MODEL;
}
```

> regex
```ts
if (/claude|anthropic/i.test(currentModel)) {
    return ANTHROPIC_SUMMARIZE_MODEL;
}

```